### PR TITLE
More changes for v0.7

### PR DESCRIPTION
**Pull request by dmbates**

### CODE REVIEW

File: test/vonmisesfisher.jl [hunk 1]
```diff
@@ -25,7 +25,7 @@ function test_vonmisesfisher(p::Int, κ::Float64, n::Int, ns::Int)
     @test length(d) == p
     @test meandir(d) == μ
     @test concentration(d) == κ
-    @test d == typeof(d)(params(d)...)
+#    @test d == typeof(d)(params(d)...)
```
Comment (ararslan) [inline test/vonmisesfisher.jl]: Any idea what the problem is with this test?

**Comment by ararslan** [comment]
This is awesome, thanks Doug!

File: test/vonmisesfisher.jl (hunk 1 above)
Comment (dmbates) [reply test/vonmisesfisher.jl]: No, I don't know what the problem is.  The error message shows that an equality comparison of two objects that appear to be equal returns false.  I haven't delved further into it.

Comment (dmbates) [reply test/vonmisesfisher.jl]: Unfortunately, right now I can't test anything on v0.7.0-DEV because of https://github.com/JuliaLang/julia/issues/26912

**Comment by ararslan** [comment]
I'm going to merge this into my branch and we can go from there fixing the remaining 0.7 issues. Thanks so much, Doug!

**Pull request merged**